package.json

```diff
@@ -48,8 +49,8 @@
     "can-reflect": "^1.6.0",
     "can-simple-map": "^4.0.0",
     "can-simple-observable": "^2.0.0",
-    "can-symbol": "^1.0.0",
-    "can-util": "^3.9.0"
+    "can-string": "0.0.5",
```

Comment:
I think this should be `"<2.0.0"`

Reply:
For can-string?

Reply:
Yes. We don’t want to pin to a specific version and we should release a 1.0 without breaking compatibility.

Reply:
Fixed!